src/components/HistoryPage.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import Toolbar from './Toolbar';
+import Logo from './Logo'
+export default class HistoryPage extends Component {
+    constructor(props) {
+        super(props)
+    
+        this.state = {
+             userHairCuts:[{date:"12.02.2020", price:50, key:1},
+                {date:"12/02/2020", price:50, key:2},
+                {date:"10/01/2020", price:50, key:3},
+                {date:"20/12/2019", price:40, key:4},
+                {date:"08/10/2019", price:40, key:5},]
```

Comment:
Why the first one is with dots and the seconds is with slashes?

Where is the hour of the haircut?